.github/workflows/build.yml

```diff
@@ -18,10 +18,14 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install -e .
```

Comment:
As I understand this, here you install the repository(?)
But you also install the plugins in line `25`, isn't this a duplication?